packages/admin-ui/src/app/participants/pending-approvals.component.ts

```diff
@@ -0,0 +1,222 @@
+import { Component, OnDestroy, OnInit } from "@angular/core";
+import { ParticipantsService } from "../_services_and_types/participants.service";
+import { MessageService } from "../_services_and_types/message.service";
+import { BehaviorSubject } from "rxjs";
+import {
+  IParticipantFundsMovement,
+  IParticipantNetDebitCapChangeRequest,
+} from "@mojaloop/participant-bc-public-types-lib";
+import { IParticipantPendingApproval } from "../_services_and_types/participant_types";
+import { UnauthorizedError } from "@mojaloop/security-bc-public-types-lib";
+
+@Component({
+  selector: "app-participants",
+  templateUrl: "./pending-approvals.component.html",
+})
+export class PendingApprovalsComponent implements OnInit, OnDestroy {
+  fundAdjustments: BehaviorSubject<
+    IParticipantPendingApproval["fundsMovementRequest"]
+  > = new BehaviorSubject<IParticipantPendingApproval["fundsMovementRequest"]>(
+    []
+  );
+  isFundAdjustmentSelectAll: boolean = false;
+  selectedFundAdjustment: IParticipantPendingApproval["fundsMovementRequest"] =
+    [];
+
+  ndcRequests: BehaviorSubject<
+    IParticipantPendingApproval["ndcChangeRequests"]
+  > = new BehaviorSubject<IParticipantPendingApproval["ndcChangeRequests"]>([]);
+  isNDCSelectAll: boolean = false;
+  selectedNDCRequest: IParticipantPendingApproval["ndcChangeRequests"] = [];
+
+  fundAdjustmentCount: number = 0;
+  ndcRequestCount: number = 0;
+
+  constructor(
+    private _participantsSvc: ParticipantsService,
+    private _messageService: MessageService
+  ) {}
+
+  async ngOnInit(): Promise<void> {
+    console.log("PendingApprovalsComponent ngOnInit");
+
+    await this.getPendingApprovalsSummary();
```

Comment:
This await is not going to work because the getPendingApprovalsSummary() is not returning a promise, same for the getPendingApprovals()